inference-engine/src/transformations/src/transformations/convert_precision.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/convert_precision.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset3.hpp>
+
+using namespace ngraph;
+
+bool ngraph::pass::ConvertPrecision::run_on_function(std::shared_ptr<ngraph::Function> f) {
+    static std::map<ngraph::NodeTypeInfo, std::function<bool(std::shared_ptr<Node>, element::Type, size_t idx)>> type_to_fuse {
+        {opset3::ShapeOf::type_info, fuse_type_to_shapeof},
```

Comment:
BTW, suggest to move this map to global const variable to make clang thread sanitizer happy.

Reply:
done.